src/batch-client/index.ts

```diff
@@ -192,6 +192,16 @@ function updateAbsoluteFolderPath(
 		return folder;
 	});
 }
+/**
+ * Return an empty string in case it's empty array or null value,
+ * else return an Array.
+ *
+ * @param value An Array or empty String
+ */
+function convertStringAndArrayValues(value: any) {
+	const result = [].concat(value).filter(Boolean);
+	return result.length ? result : '';
```

Comment:
i think graphql will expect an empty array instead of empty string

Reply:
Covered by https://github.com/Zimbra/zm-api-js-client/pull/376/files#r444237374